### ISSUE TITLE
Duplicate result popup added to iframe

### ISSUE THREAD

**Issue by ABeard89** [ashchan/safarikai#17]
When hovering over a word, result popup also appears in child `iframe` documents.

I noticed this when viewing a page with several ads embedded via iframes.
Seeing multiple result boxes was very distracting.

I used [W3Schools' iframe test page](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_iframe) to reproduce this.

HTML typed into the editor will show within an iframe.
When hovering over a word (that I type into the test editor), a Safarikai popup is added to the main test result's iframe, as well as any iframes within that one.

In short, popups are added **_recursively_** to all iframes within the document hierarchy.

OS Version: 10.13.5 (17F77)
Safari Version: 11.1.1 (13605.2.8)

**Comment by ashchan** [comment]
@ABeard89 Yes this is very annoying. I remembered I fixed a similar iframe issue long time ago. Do you have any other websites that having this problem? I wonder if W3School's playground has an exceptional structure that causes this.

**Comment by ABeard89** [comment]
@ashchan After noticing it there, then reproducing it on W3Schools, I assumed it would be an issue for most, if not all, iframes.

**Comment by ABeard89** [comment]
@ashchan FYI: I also reproduced it just now on Stack Overflow.

I inserted some Japanese into the page's HTML, hovered over it, and saw the extra popup in their ad's iframe. So, I guess it must be a common iframe setup if Stack Overflow also does it.

**Comment by ashchan** [comment]
@ABeard89 Thanks for the research.

Current development of this plugin is stale that I'm not sure how to update and roll out fix. Apple has changed their way of Safari plugin development, but unfortunately another branch of effect to migrate is too old and doesn't have much progress.

**Comment by apparition47** [comment]
@ABeard89 
I have a PR. Thanks for the detailed breakdown.
